src/org/spdx/htmltemplates/LicenseHTMLFile.java

```diff
@@ -61,9 +61,14 @@
 	 */
 	public static class FormattedUrl {
 		String url;
```

Comment:
@tjasmith Can you add a comment on the expected format of the license crossref string